docs/modules_management.rst

```diff
@@ -0,0 +1,194 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Modules Management
+==================
+
+Airflow allows you to use your own Python modules in the DAG and in the Airflow configuration. The following article
+will describe how you can create your own module so that Airflow can load it correctly, as well as diagnose problems
+when modules are not loaded properly.
+
+This article is the last one for you if you need to adapt Airflow to the needs of your organization.
```

Comment:
Is it really the last one :)? I am not sure if that sentence adds any value here :)

Reply:
I've removed it.